app/models/zillow.rb

```diff
@@ -19,22 +19,32 @@ def initialize(attrs)
     @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
     @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
     @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
+    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
+    if attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
```

Comment:
I think it should be fixed now....

2016-02-02 14:45 GMT-08:00 Bookis Worthy <notifications@github.com>:

> In app/models/zillow.rb
> <https://github.com/smurfik/auction_house/pull/23#discussion_r51649171>:
>
> > @@ -19,22 +19,32 @@ def initialize(attrs)
> >      @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
> >      @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
> >      @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
> > +    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
> > +    if attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
>
> This has the same problem, since ["lastSoldPrice"] is nil, calling
> ["__content__"] on it gives an error, you want to leave ["__content__"]
> off of the conditional.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smurfik/auction_house/pull/23/files#r51649171>.
>



-- 
Karen Hambro